src/java.base/share/classes/sun/util/locale/provider/LocaleServiceProviderPool.java

```diff
@@ -148,11 +148,11 @@ private AllAvailableLocales() {
     }
 
     /**
+     * {@return a stream of the available locales for all provider classes}
+     *
      * Returns a stream of available locales for all the provider classes.
      * This stream is constructed from all the locales
      * that are provided by each provider, including the JRE.
-     *
-     * @return a stream of the available locales for all provider classes
```

Comment:
Seems weird to have the @return where you moved it to

Reply:
From what I saw in other methods, `@return ...` goes on the bottom (like the line I removed), but the javadoc 16 tag `{@return ... }` goes on the top.

Reply:
OK but then we should be consistent and update all of the methods to use the new tag or  revert the change for consistency

Reply:
Makes sense, will look at both Locale and LocaleServiceProviderPool and update the methods that are applicable for {@return ..}. Thanks Lance

Reply:
That's getting a bit far afield of the original issue; a separate issue might be a better way to address those cases. $.02

Reply:
Hi Lance and Roger, I have switched the tag {@return to @return for consistency. And like you proposed Roger, I will open a separate issue to address converting the tags.

Reply:
> Hi Lance and Roger, I have switched the tag {@return to @return for consistency. And like you proposed Roger, I will open a separate issue to address converting the tags.

Thank you Justin